test/integration/admin_login_test.rb

```diff
@@ -47,13 +47,15 @@ def setup
   test "registered admin can see create category on menu page" do
     skip
     ApplicationController.any_instance.stubs(:current_user).returns(user)
-    visit items_path
-    click_link_or_button "Create Category"
-    within(".category") do
-      assert page.has_content?("Create New Category")
+    visit root_path
+    click_link_or_button('Admin Dashboard')
+    assert admins_path, current_path
+    within ('#admin_header') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.